### ISSUE TITLE
Need in game 'E' key to float over pickupable things

### ISSUE THREAD

**Issue by johnkslg** [StrangeLoopGames/EcoIssues#2750]

**Comment by MarjieAVolk** [comment]
Yes, this is what I want for everything.  Tell you how to use it

**Comment by Nanders** [comment]
I think this is a good idea.

Maybe have it appear after a timer and only if its within interact distance? To prevent spam and make it less annoying to people who know how to interact.

Alternatively just have it as a tickable option in playerprefs
